docs/blog-post.md

```diff
@@ -0,0 +1,286 @@
+<h1><b>How to make akka serialization bulletproof</b></h1>
+
+![bulletproof shield](https://images.unsplash.com/photo-1561156772-a44477f220a5?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1974&q=80)
+<i>bulletproof shield from [Unsplash](https://images.unsplash.com/photo-1561156772-a44477f220a5) by [NIFTYART](https://unsplash.com/@niftyartofficial1_)</i>
```

Comment:
There won't be any convenient place to put these credits in Medium anyway ☹️ I suppose Unsplash does not **require** acknowledging authorship, just **recommends**?

Reply:
(huh, maybe setting alt-text for splash photo is possible on Medium at least, to be checked)

Reply:
Ok, I'll try to add these credits on Medium after this PR - we'll see if it would be possible. If not - the picture will go without any credits.

Reply:
You are right - unsplash does not require credits, but I think that it would be nice to add them.